ipaplatform/debian/paths.py

```diff
@@ -58,7 +58,7 @@ class DebianPathNamespace(BasePathNamespace):
     SYSCONFIG_IPA_DNSKEYSYNCD = "/etc/default/ipa-dnskeysyncd"
     SYSCONFIG_IPA_ODS_EXPORTER = "/etc/default/ipa-ods-exporter"
     SYSCONFIG_KRB5KDC_DIR = "/etc/default/krb5-kdc"
-    SYSCONFIG_NAMED = "/etc/default/bind9"
+    SYSCONFIG_NAMED = "/etc/default/named"
```

Comment:
Could it be conditional?
I mean this change could break the systems having BIND < 9.16 if any.

Reply:
I don't think that's necessary, then again this could be dropped until there's actual support for 9.16

Reply:
I've dropped this from the set, there are other changes needed but I'll let master use 9.11 until the test suite passes